applications/ecallmgr/src/ecallmgr_util.erl

```diff
@@ -39,7 +39,7 @@
 
 -include("ecallmgr.hrl").
 
--define(HTTP_GET_PREFIX, "${http_get(").
+-define(HTTP_GET_PREFIX, "http_cache://").
```

Comment:
@jamesaimonetti 
tested MOH, ringback, vm, conferences... Look like it all works good.